bolt_store.go

```diff
@@ -27,7 +27,7 @@ var (
 // a LogStore and StableStore.
 type BoltStore struct {
 	// conn is the underlying handle to the db.
-	conn *bolt.DB
+	Conn *bolt.DB
```

Comment:
Do we need to make this public given that the transition function is in the same package? Do we want calling code to be able to access the bolt DB directly? It feels like a strange API to expose internals to the consumer like this unless there is a really good reason and clear docs on what is expected/supported behaviour etc.

Reply:
Oh wait, I think I get it - it's because v2 and v1 are separate packages so can't do this another way.

Could we just expose it another way say with an accessor method like:

```go
   func (b *BoltStore) InternalUseOnlyAccessBoltDB() *bolt.DB {}
```

With an appropriately scary doc comment?

Reply:
From my understanding currently I need that to be public so that we can read it from [Bbolt here ](https://github.com/hashicorp/raft-boltdb/pull/23/files#diff-cd3e2fa22ef4a74a78b671e33b9450c71b147125fde4996e3bd395d9f3da14e5R277) - so no matter where I call it from I will need to connect to the other side (I think)

Reply:
Yeah I also realise you mentioned this in the description so shame on me for not reading closely enough 😄

Reply:
Accessor method is a great idea! I'll add that.